lib/notifier.js

```diff
@@ -234,7 +234,9 @@ function alertSubject (alertRec) {
   if (alerts.length === 0) return null
 
   const alertTypes = _.map(codeDisplay, _.uniq(_.map('code', alerts))).sort()
-  return '[Lamassu] Errors reported: ' + alertTypes.join(', ')
+  const deviceNames = _.values(alertRec.deviceNames)
+
+  return '[Lamassu] Errors reported: ' + alertTypes.join(', ') + ' (' + deviceNames.join(', ') + ')'
```

Comment:
This isn't quite according to the Trello card spec. The idea was to have each alertType individually show which machines it's affecting, not to list all machines affected by all alerts in the end.